src/fairchem/core/models/escn/escn_exportable.py

```diff
@@ -0,0 +1,876 @@
+    GaussianSmearing,
+    LinearSigmoidSmearing,
+    SigmoidSmearing,
+    SiLUSmearing,
+)
+
+with contextlib.suppress(ImportError):
+    from e3nn import o3
+
+
+@registry.register_model("escn_export")
+class eSCN(nn.Module):
+    """Equivariant Spherical Channel Network
+    Paper: Reducing SO(3) Convolutions to SO(2) for Efficient Equivariant GNNs
+
+
+    Args:
+        use_pbc (bool):         Use periodic boundary conditions
+        use_pbc_single (bool):         Process batch PBC graphs one at a time
+        regress_forces (bool):  Compute forces
+        otf_graph (bool):       Compute graph On The Fly (OTF)
+        max_neighbors (int):    Maximum number of neighbors per atom
+        cutoff (float):         Maximum distance between nieghboring atoms in Angstroms
+        max_num_elements (int): Maximum atomic number
+
+        num_layers (int):             Number of layers in the GNN
+        lmax (int):                   maximum degree of the spherical harmonics (1 to 10)
+        mmax (int):                   maximum order of the spherical harmonics (0 to lmax)
+        sphere_channels (int):        Number of spherical channels (one set per resolution)
+        hidden_channels (int):        Number of hidden units in message passing
+        num_sphere_samples (int):     Number of samples used to approximate the integration of the sphere in the output blocks
+        edge_channels (int):          Number of channels for the edge invariant features
+        distance_function ("gaussian", "sigmoid", "linearsigmoid", "silu"):  Basis function used for distances
+        basis_width_scalar (float):   Width of distance basis function
+        distance_resolution (float):  Distance between distance basis functions in Angstroms
+    """
+
+    def __init__(
+        self,
+        use_pbc: bool = True,
+        use_pbc_single: bool = False,
+        regress_forces: bool = True,
+        otf_graph: bool = False,
+        max_neighbors: int = 40,
+        cutoff: float = 8.0,
+        max_num_elements: int = 90,
+        num_layers: int = 8,
+        lmax_list: list[int] = (4), # list of 1, for backward compat only right now,
+        mmax_list: list[int] = (2), # list of 1, for backward compat only right now,
+        sphere_channels: int = 128,
+        hidden_channels: int = 256,
+        edge_channels: int = 128,
+        num_sphere_samples: int = 128,
+        distance_function: str = "gaussian",
+        basis_width_scalar: float = 1.0,
+        distance_resolution: float = 0.02,
+        resolution: int | None = None,
+    ) -> None:
+        super().__init__()
+
+        import sys
+
+        if "e3nn" not in sys.modules:
+            logging.error("You need to install the e3nn library to use the SCN model")
+            raise ImportError
+
+        self.regress_forces = regress_forces
+        self.use_pbc = use_pbc
+        self.use_pbc_single = use_pbc_single
+        self.cutoff = cutoff
+        self.otf_graph = otf_graph
```

Comment:
currently this model assumes `otf_graph=False`. If this is what's going to be in the final version I suggest raise an error if passed `otf_graph=True`.

Reply:
ya let me remove this actually